arex-storage-web-api/src/main/java/com/arextest/storage/mock/internal/matchkey/impl/DatabaseMatchKeyBuilderImpl.java

```diff
@@ -168,7 +170,10 @@ private List<byte[]> dbMockKeyBuild(Mocker databaseMocker) {
     byte[] dbNameBytes = CacheKeyUtils.toUtf8Bytes(dbName);
```

Comment:
At this time, opName has been spliced with dbname and body. Why can't it be obtained directly through operationname?